### DISCUSSION TITLE
use bookmarklets via a proxy and iframe (works on v122 you cant use stuff like ext-launcher)

### DISCUSSION THREAD

**Discussion by stuartcatt** [3kh0/ext-remover#1033] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
I have noticed that the iframe.contentwindow.eval() can be used to effect other parts of the same site such as example.com/codeexec making edits to example.com/home in an iframe so I have used a proxy so I am able to use the execution in an iframe on all sites (Google is blocked since it knows its a proxy) try it out! use alt+j to run code and alt+L to change the URL https://www.zerotoone.com.br/wp-content/plugins/super-links/application/helpers/super-links-proxy.php?https://stuartcatt.github.io/ss/ this has taken me not to long to figure out

**Comment by stuartcatt** [comment]
this cant be used to do stuff like ext-launcher but you can use it for fun bookmarklets like edpuzzle skippers or auto clickers

**Comment by helloIlovegimkit** [comment]
incase your school bolcked it (misspelled incase you get bolcked)

https://www.mediafire.com/file/6akarv6inup3s1j/bolcked.html/file
